README.md

```diff
@@ -116,6 +116,7 @@ In either case, upgrading to module version `v1.0.0` will trigger a recreation o
 |------|-------------|:----:|:-----:|:-----:|
 | basic\_auth\_password | The password to be used with Basic Authentication. | string | `""` | no |
 | basic\_auth\_username | The username to be used with Basic Authentication. An empty value will disable Basic Authentication, which is the recommended configuration. | string | `""` | no |
+| database\_encryption | Application-layer Secrets Encryption settings. Example:   database_encryption = [{     state = "ENCRYPTED",     key_name = "projects/my-project/locations/global/keyRings/my-ring/cryptoKeys/my-key"   }] | list | `<list>` | no |
```

Comment:
Shouldn't the default here be `[]`, not `<list>`?

Reply:
This document is generated using `make generate_docs`. I would prefer not to modify anything that is generated by that task.

Reply:
Can we raise a bug about this? Doesn't seem like expected behavior.

Reply:
Raise a bug on `terraform-docs` or a bug on `make generate_docs`? You can see `master_authorized_networks_config` has the same behavior. 

If you look at https://github.com/terraform-google-modules/terraform-google-project-factory you will see that `shared_vpc_subnets` has the same format.

Reply:
`terraform-docs`. This is ok to merge as-is, but it's worth considering raising a bug/inquiry upstream.